lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -27,12 +36,36 @@ $badge-min-height: 20px;
 		@include label-xs-default-bold-uppercase;
 	}
 
+	&__icon {
+		align-items: center;
+		display: flex;
+		height: $icon-xxs;
+		margin-right: $space-xxs;
+		width: $icon-xxs;
+	}
+
 	&.-neutral {
 		background-color: $color-neutral-background-medium;
 
 		.badge__label {
 			color: $color-neutral-text;
 		}
+
+		.badge__icon {
+			color: $color-neutral-text;
```

Comment:
UX hate when we use `text` token on icons

Reply:
Git you, changed to `$color-neutral-icon` and `color: $color-primary-icon`